### PR TITLE
Search view styles

### PR DESCRIPTION
**Pull request by Cindyzhang977**
image

### CODE REVIEW

File: frontend/src/traveler.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....92,17.94,0,0,35.88-6.52,44-1.63,0,0-27.72,9.78-26.09,16.31,0,0,48.92,0,55.45,17.94,0,0-50.55,6.52-53.82,11.42,0,0,58.71,42.4,62,53.82,0,0-60.34-35.88-70.12-35.88s44,52.18,40.77,65.23C755.1,366.14,711.07,314,699.65,317.22Z" transform="translate(-196.89 -84.52)" fill="#4c8bf5"/></g><g opacity="0.3"><path d="M821.55,559H806.28s-4.77-166.92,15.26-218.43h3.82S812,529.43,821.55,559Z" transform="translate(-196.89 -84.52)" fill="#4c8bf5"/><path d="M811.05,332s-12.4,34.34-6.68,49.6c0,0,8.58-31.48,18.6-36.25s20.51,36.25,20.51,36.25,5.72-25.75-4.77-46.74c0,0,21,11.45,31.48,33.38,0,0,1.91-14.31-22.89-43.88,0,0,21.94,5.72,36.25,19.08,0,0,4.77-10.49-24.8-27.66,0,0,3.82-12.4,26.71-6.68,0,0,0-16.22-36.25-13.35,0,0,0-10.49,23.85-8.58,0,0,0-13.35-27.66-8.58,0,0,5.72-8.58,9.54-10.49,0,0-7.63-1-13.35,3.82,0,0,1-17.17-13.35-23.85l-1.91,23.85s-16-18.89-12.4-15.26c1,1,3.82,12.4,2.86,18.12,0,0-16.22-13.35-36.25,1,0,0,24.8,1.91,28.62,10.49,0,0-21-3.82-25.75-1,0,0,16.22,5.72,15.26,9.54,0,0-28.62,0-32.43,10.49,0,0,29.57,3.82,31.48,6.68,0,0-34.34,24.8-36.25,31.48,0,0,35.29-21,41-21S776.71,353,778.62,360.6C778.62,360.6,804.38,330.08,811.05,332Z" transform="translate(-196.89 -84.52)" fill="#4c8bf5"/></g></svg>
```
Comment (genedwards) [inline frontend/src/traveler.svg:1]: looks great!

it also looks like you got this from [somewhere,](https://github.com/search?l=SVG&q=3d2e1720-bd24-4bc0-a54b-69e5d5c1743b&type=Code)
what kind of license is it? does the license require attribution?

Comment (Cindyzhang977) [reply frontend/src/traveler.svg:1]: https://undraw.co/ has free open source illustrations :)

Comment (genedwards) [reply frontend/src/traveler.svg:1]: oh, that's awesome

File: frontend/src/saved-trips/SavedTrips.module.css [hunk 2]
```diff
@@ -1,17 +1,20 @@
 .container {
+  height: 42vh;
   margin: auto;
   margin-top: 48px;
   width: 80%;
+  overflow-y: scroll;
```
Comment (mecolmg) [inline frontend/src/saved-trips/SavedTrips.module.css:6]: nit: auto would be better. scroll makes the scrollbar always visible on certain browsers, while auto will only show it when necessary

**Pull request merged**